### PR TITLE
additional guidelines on Perl skeleton recipes

### PR DESCRIPTION
**Pull request by jvolkening**
...based on my experiences submitting a recipe for a CPAN module.

### CODE REVIEW

File: docs/source/guidelines.rst [hunk 1]
```diff
@@ -351,6 +351,14 @@ If the recipe was created with ``conda skeleton cpan``, the tests are
 likely sufficient. Otherwise, test the import of modules (see the
 ``imports`` section of the ``meta.yaml`` files in above examples).
 
+Additionally, if the recipe was created with ``conda skeleton cpan``, several modifications
+are necessary to satisfy bioconda policies:
+
+- remove the ``bld.bat`` script
+- remove the ``source/fn`` entry in ``meta.yaml``
+- the ``requirements/build`` keyword in ``meta.yaml`` should be changed to
```
Comment (bgruening) [inline docs/source/guidelines.rst]: Hu, is this still the case with the latest conda version?

Comment (bgruening) [reply docs/source/guidelines.rst]: ping @jvolkening

Comment (epruesse) [reply docs/source/guidelines.rst]: As of 4.5.11, yes to all three. It also creates a version subdir.

Comment (epruesse) [reply docs/source/guidelines.rst]: Also generates `version: {{version}}` which is broken (`0.80` will be parsed as `0.8` by yaml).

**Pull request merged**